src/app/App.tsx

```diff
@@ -1,7 +1,19 @@
+import Footer from "@/components/ui/footer";
 import { AppRouter } from "@/features/router";
 
 function App() {
-  return <AppRouter />;
+  return (
+    <div className="flex flex-col min-h-screen">
+      {/* AppRouter handles the main content */}
+      <div className="flex-1">
+        <AppRouter />
+      </div>
+
+      {/* Footer sticks to the bottom */}
+      <Footer />
```

Comment:
Good start. Nice showing it here temporarily, Once my PR goes in. i will use the footer in the appropriate file.